### ISSUE TITLE
(frida:13397): GLib-GIO-WARNING **: 00:04:46.046: _g_dbus_worker_do_read_cb: error determining bytes needed: Blob indicates that message exceeds maximum message length (128MiB

### ISSUE THREAD

**Issue by GetCode747** [sensepost/objection#516]
what is it?

**Comment by CDuPlooy** [comment]
This happens if you try to transfer more than 128MiB using your agent (or the objection agent). You'll need to add some more context to this issue, like what you were trying to do at the time.

**Comment by GetCode747** [comment]
> 
I'm just using the memory dump app.
"memory dump all ./file"

**Comment by CDuPlooy** [comment]
Yeah so the page it's reading is larger than 128MiB, I've been meaning to have a look at that at some point - in the mean time you can use r2frida, which I believe can do the same thing. I'll refer to this issue if I create a PR for this.

**Comment by GetCode747** [comment]
> Yeah so the page it's reading is larger than 128MiB, I've been meaning to have a look at that at some point - in the mean time you can use r2frida, which I believe can do the same thing. I'll refer to this issue if I create a PR for this.

thank you for your help,I'm going to try,I will continue to update the issues I encounter.

**Comment by CDuPlooy** [comment]
You can try this PR in the mean time #520, I may have made an off by one error somewhere so some additional eyes on it would be appreciated.

**Comment by GetCode747** [comment]
> You can try this PR in the mean time #520, I may have made an off by one error somewhere so some additional eyes on it would be appreciated.

Thanks for your response.

**Comment by Gitdork** [comment]
sorry ,i still don't understand ... where or which file should i change?
